crates/meroctl/src/cli/context/create.rs

```diff
@@ -39,25 +44,45 @@ impl CreateCommand {
 
         let client = Client::new();
 
-        match self {
+        // Manually check for exclusivity between application_id and watch
+        if self.application_id.is_some() && self.watch.is_some() {
```

Comment:
Throws strange error 
```
error: the argument '--watch <WATCH>' cannot be used with one or more of the other specified arguments

Usage: meroctl context create [OPTIONS] [METADATA]...
```

if `app_id` and watch are exclusive when I provide `watch` and `context`-id.